utils/install-app.js

```diff
@@ -139,6 +141,107 @@ function addToServer(appDir, installDir) {
   }
 }
 
+function copyRecognizers(appDir, appId, appVers) {
+  let recognizers;
+  let recognizersKeys;
+  let configRecognizers;
+  let configLocation;
+
+  try { // Get recognizers in a plugin's appDir/config/xxx location
+    recognizers = JSON.parse(fs.readFileSync(path.join(appDir, "config/recognizers", appId))).recognizers;
```

Comment:
you need to read all files in that folder, not just the one with the same name as your appId
App A can include recognizer files for both A and B.
Put this entire thing in a loop, and read each file you find.